archeo/style.css

```diff
@@ -224,3 +224,13 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 	height: auto;
 	max-width: 100%;
 }
+
+/*
+ * Custom font size at small screens for navigation.
+ * Possible future fix: https://github.com/WordPress/gutenberg/pull/33543
+ */
+@media (max-width: 599px) {
+	.wp-block-navigation__responsive-container-content li {
```

Comment:
I noticed this selector doesn't work in the editor for non-standard links. For example, I have Submenu and a Home link, and the large font size isn't applying: 
<img width="1904" alt="Screen Shot 2022-03-09 at 2 12 32 PM" src="https://user-images.githubusercontent.com/5375500/157514605-3f786a8d-a89c-4d38-a850-f5bf0b2d394a.png">

It makes me wonder if adding this CSS is worth it, or if it's going to cause more confusion if a user changes this navigation's font size in the editor and previews it. 

Another thought — could we create a custom CSS variable that uses a responsive type size and apply it to the navigation block [here](https://github.com/Automattic/themes/blob/trunk/archeo/theme.json#L140). That way any user modifications to the font-size would still take precedence.

Reply:
> could we create a custom CSS variable that uses a responsive type size and apply it to the navigation block [here](https://github.com/Automattic/themes/blob/trunk/archeo/theme.json#L140).

Oh that's a fine idea.

Reply:
I think the CSS variable with a responsive type size is a great idea, however, I'm not sure how to do this where the font size is bigger on smaller screens and smaller on larger screens.

For now, I've gone with setting the larger font size for the overlay in CSS (we were already doing this, I missed it when I was working on this PR!) and setting the default font size in theme.json.